packages/evm/src/__test__/fungible.test.ts

```diff
@@ -25,7 +25,7 @@ const TRANSFER_PARAMS = {
     caip19: '0x11',
   },
   amount: parseEther('10').toBigInt(),
-  destinationAddress: '0x98729c03c4D5e820F5e8c45558ae07aE63F97461',
+  recipientAddress: '0x98729c03c4D5e820F5e8c45558ae07aE63F97461',
 };
 
 const MOCKED_CONFIG = {
```

Comment:
used in fungible and non-fungible test cases, so it can be moved somewhere to avoid code duplication